modules/project_cleanup/function_source/main.go

```diff
@@ -175,6 +186,21 @@ func checkIfTagKeyShortNameExcluded(shortName string, excludedTagKeys []string)
 	return false
 }
 
+///r
+func checkIfCaiFeedsShortNameIncluded(shortName string, IncludedFeeds []string) bool {
```

Comment:
this function should be used to check if the current feed should be deleted.
The logic is not correct for that.
the function is not used in the code, it should be used ti decide if the feed will be deleted or not